lib/services/sku-pack-service.js

```diff
@@ -43,6 +45,24 @@ function skuPackServiceFactory(
     }
 
     /**
+     * Given a nodeId, return the prioritized scope order
+     * @param  {string}     nodeId
+     */    
+     SkuPackService.prototype.setupScope = function(nodeId) {
+        var defaultScope = ['global'];
+        if(typeof(nodeId) === 'undefined')  {
```

Comment:
Why not just `if(!nodeId)`? This way if nodeId is null the conditional still hits.